apps/rush-lib/src/cli/utilities/ChangelogGenerator.ts

```diff
@@ -92,10 +86,11 @@ export default class ChangelogGenerator {
         }
 
         const changelog: IChangelog = ChangelogGenerator._getChangelog(project.packageName, project.projectFolder);
+        const isLockstepped: boolean = !!project.versionPolicy && project.versionPolicy.isLockstepped;
```

Comment:
>!! [](start = 39, length = 2)

Neither is this one.

Reply:
This one is actually needed. Other compiler would complain.

---
In reply to: [155398616](https://github.com/Microsoft/web-build-tools/pull/462#discussion_r155398616) [](ancestors = 155398616)